python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -1101,9 +1102,10 @@ def test_coercion_to_float(lmdb_version_store_string_coercion):
 
     assert df["col"].dtype == np.object_
 
-    with pytest.raises(ArcticNativeNotYetImplemented):
-        # Needs pickling due to the obj column
-        lib.write("test", df)
+    if sys.platform != "win32":  # SKIP_WIN Windows always uses dynamic strings
```

Comment:
GitHub issue reference unless we don't ever want to support static strings on Windows?